### ISSUE TITLE
Font not loading 

### ISSUE THREAD

**Issue by Amarasekara** [photopea/photopea#3812]
font's not loading and when fonts change software stucking

**Comment by photopea** [comment]
If you are using a browser or an extension with a bug (e.g. your browser is not capable of saving a file), there is nothing a website can do about it :(

**Comment by shurvir49** [comment]
> So the problem is with loading a font? Do you see any errors in a web console?

No. actually I didn't started console while happends this. also console shows next occur error after opening console. while We got error then open console, then nothing shows.

**Comment by marcvi1a** [comment]
"Londrina Solid" not loading either. Tried in Incognito and different browsers as well but error persists.

**Comment by camoody1** [comment]
> "Londrina Solid" not loading either. Tried in Incognito and different browsers as well but error persists.

I was just able to load all variations of this font without issue in a regular Edge browser tab. Not sure if the issue is just intermittent or local to your machine. Is it still not working for you?

**Comment by marcvi1a** [comment]
Still not working, here a video showing step by step https://www.loom.com/share/8d4951c201414b03a5281c90d64b58eb

Same behavior on incognito and Firefox.

**Comment by marcvi1a** [comment]
On Firefox steps are:
1. Open photopea.com
2. Go to Edit, then Local Storage
4. LondrinaSolid-Black.ttf and LondrinaSolid-Regular.ttf are there
5. Select Type Tool
6. Click dropdown fonts
7. "Find" Londrina
8. Only standard "Londrina" appears, not LondrinaSolid

**Comment by photopea** [comment]
To aplly a specific font to your text, you must **click on the font in a font list** (click on the name or on the thumbnail).

You have opened the font menu many times during your 2-minute video, you even typed the name of the font, but you never clicked the font. You need to click on it, as shown below.

![image](https://user-images.githubusercontent.com/15955558/215030173-9898f1b6-c9a9-4e16-8983-6547aa0723e6.jpg)

**Comment by camoody1** [comment]
That font is available in Photopea by default. You should not have to load it from local storage. After selecting the Londrina font, you need to select your preferred version of the font from the droplist on the top toolbar. There are eight different versions of the font to choose from.

**Comment by marcvi1a** [comment]
Oh, my bad. I never saw the second dropdown with the font type and assumed Londrina Outline was the only one available. Thanks!
